dspace-server-webapp/src/test/java/org/dspace/app/rest/ResourcePolicyRestRepositoryIT.java

```diff
@@ -2202,6 +2202,518 @@ public void patchAddNameTest() throws Exception {
                 hasJsonPath("$.name", is(name)))));
     }
 
+    @Test
+    public void patchAddActionTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        EPerson eperson1 = EPersonBuilder.createEPerson(context)
```

Comment:
Here already defined user in field "eperson" of base class can be used without creating a new one